src/System.Security.Cryptography.Xml/tests/RSAKeyValueTest.cs

```diff
@@ -96,7 +96,16 @@ public void LoadXml_InvalidXml(string xml)
             xmlDocument.LoadXml(xml);
 
             RSAKeyValue rsa = new RSAKeyValue();
-            Assert.Throws<CryptographicException>(() => rsa.LoadXml(xmlDocument.DocumentElement));
+
+            // FormatException exception because desktop does not
+            // check if Convert.FromBase64String throws
+            // Related to: https://github.com/dotnet/corefx/issues/18690
```

Comment:
You and @karelz should come up with a plan, since the issue number is just us doing what he asked :smile:.